controls/radheatmap/row-column-headers.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Row/Column Headers
+page_title: Row/Column Headers
+description: Row/Column Headers
+slug: radheatmap-row-column-headers
+tags: tooltip, heatmap, row, columnheader, celltooltip, heatmaprowheadersettings, heatmapcolumnheadersettings
+published: True
+position: 5
+---
+
+#  Row/Column Headers
+
+The Row/Column headers of the RadHeatMap can be further customized through  the __RowHeaderSettings__ and __ColumnHeaderSettings__ properties. 
+
+## Row Headers
+
+The customization of the row headers is possible through  the __RowHeaderSettings__ property of the RadHeatMap. This property is of type __HeatMapRowHeaderSettings__ object. Except for exposing the common properties for styling the labels, the __HeatMapRowHeaderSettings__ expose an additional one to change the position of the label to the right side of the RadHeatMap. 
```

Comment:
"_Except for exposing the common properties for styling the labels, the __HeatMapRowHeaderSettings__ expose an additional one to change the position of the label to the right side of the RadHeatMap._"

Here it is not clear what are the common label styling properties. Add information about those.
Also, the word you can replace the second appearance of the word "expose" with something else.